march_hardware/include/march_hardware/imotioncube/motor_controller.h

```diff
@@ -0,0 +1,39 @@
+// Copyright 2019 Project March.
+
+#include "actuation_mode.h"
+#include "motor_controller_state.h"
+#include <string>
+
+namespace march
+{
+class MotorController
+{
+public:
+  virtual double getAngleRadAbsolute() = 0;
+  virtual double getAngleRadIncremental() = 0;
+  virtual double getVelocityRadAbsolute() = 0;
+  virtual double getVelocityRadIncremental() = 0;
+  virtual int16_t getTorque() = 0;
+  virtual MotorControllerStates getStates() = 0;
+
+  virtual ActuationMode getActuationMode() const = 0;
+  virtual uint16_t getSlaveIndex() const = 0;
```

Comment:
What if a motor controller isn't a slave or doesn't use the master-slave structure.

Reply:
Added a docstring to explain